api/SVGAnimatedBoolean.json

```diff
@@ -14,10 +14,10 @@
             "version_added": "≤18"
           },
           "firefox": {
-            "version_added": true
+            "version_added": "3"
```

Comment:
You gotta love an animated boolean! I'm sure there are cases where it makes some sense to animate a boolean, but at first glance it's a funny concept :)